src/nyc_trees/apps/users/templates/groups/detail.html

```diff
@@ -67,6 +67,14 @@ <h4 class="section-heading">Description</h4>
                 </div>
             </div>
         {% endif %}
+        <div class="description block item">
+            <div class="row">
+                <div class="col-xs-8">
+                    <h5>Affiliation</h5>
+                    <div class="h6">{{group.affiliation}}</div>
+                </div>
+            </div>
+        </div>
```

Comment:
I'm torn on this but inclined to say it should be hidden if not completed. I'll defer to @jfrankl though.